node/sync/src/block_sync.rs

```diff
@@ -420,15 +421,13 @@ impl<N: Network> BlockSync<N> {
 
     /// Updates the block locators and common ancestors for the given peer IP.
     /// This function checks that the given block locators are well-formed, however it does **not** check
-    /// that the block locators are consistent the peer's previous block locators or other peers' block locators.
+    /// that the block locators are consistent with the peer's previous block locators or other peers' block locators.
     pub fn update_peer_locators(&self, peer_ip: SocketAddr, locators: BlockLocators<N>) -> Result<()> {
         // If the locators match the existing locators for the peer, return early.
         if self.locators.read().get(&peer_ip) == Some(&locators) {
             return Ok(());
         }
 
-        // Ensure the given block locators are well-formed.
```

Comment:
Why was this removed? Maybe you can leave a comment here that states why we do not need to check for the validity of the block locators.

Reply:
Yes, I updated the doc comment for the function.